docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,66 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+.. toctree::
+  :hidden:
+
+The Scala bindings is a client implementation of the *Ledger API*
+based on `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_.
+
+Introduction
+************
+
+The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using Scala programming language.
+
+There are two main components:
+
+- The Akka Streams based API
+    The API to send commands to the ledger and receive transactions back.
+
+- Scala codegen
+    DAML to Scala code generator. Generates Scala classes from DAML models. The generated Scala code provides a type safe way of creating contracts (create command) and exercising contract choices (exercise command).
+
+The reader must be familiar with:
+
+- :doc:`DAML language </daml/reference/index>`
+- :doc:`Ledger API </app-dev/grpc/index>`
+- `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_
+- `Scala programming language <https://www.scala-lang.org>`_
+
+Getting started with SBT
+************************
+
+Setting up dependencies in ``build.sbt``:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:dependencies>
+   :end-before: // </doc-ref:dependencies>
+
+There are two modules in the example below:
+
+- ``scala-codegen``
+    This modules contains generated Scala ADTs.
+- ``application``
+    This is the application code that makes use of the generated Scala ADTs.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:modules>
+   :end-before: // </doc-ref:modules>
+
+``scala-codegen`` module uses the following function to generate Scala ADTs from a DAR file.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:generate-scala>
+   :end-before: // </doc-ref:generate-scala>
+
+Here is the entire `build.sbt file <https://github.com/digital-asset/daml/blob/master/language-support/scala/examples/quickstart-scala/build.sbt>`_
+
+Example code
+************
+
+TODO: Explain ADTs here
```

Comment:
This is missing?

Reply:
yep, it is still WIP... it is a darf PR :)